fuzzers/libafl-fuzz/Cargo.toml

```diff
@@ -6,8 +6,8 @@ edition = "2021"
 [dependencies]
 clap = { version = "4.5.8", features = ["derive", "env"] }
 env_logger = "0.11.3"
-libafl = { path = "../../libafl", features = ["std", "derive", "track_hit_feedbacks", "clap"]}
-libafl_bolts = { path = "../../libafl_bolts", features = ["std"]}
+libafl = { path = "../../libafl", features = ["std", "derive", "track_hit_feedbacks", "clap", "errors_backtrace"]}
```

Comment:
someone (tm) should benchmark if this degrades performance - else it should be default

Reply:
errors backtrace?

Reply:
Yes